smoke-tests/src/androidTest/backend/functions/functions/package.json

```diff
@@ -10,12 +10,12 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node": "12"
+    "node": "19"
```

Comment:
I'm assuming this is fine since it's a test?

Reply:
I think the same